doc/shell/shell_guide.md

```diff
@@ -0,0 +1,844 @@
+default.ha.ha_port_id=19909
+
+env.104.ssh.host=192.168.1.104
+env.104.ssh.port=22
+env.104.ssh.user=shell
+env.104.ssh.pwd=PASSWORD
+
+env.105.ssh.host=192.168.1.105
+env.105.ssh.port=22
+env.105.ssh.user=shell
+env.105.ssh.pwd=PASSWORD
+
+env.106.ssh.host=192.168.1.106
+env.106.ssh.port=22
+env.106.ssh.user=shell
+env.106.ssh.pwd=PASSWORD
+
+env.107.ssh.host=192.168.1.107
+env.107.ssh.port=22
+env.107.ssh.user=shell
+env.107.ssh.pwd=PASSWORD
+
+env.108.ssh.host=192.168.1.108
+env.108.ssh.port=22
+env.108.ssh.user=shell
+env.108.ssh.pwd=PASSWORD
+
+env.109.ssh.host=192.168.1.109
+env.109.ssh.port=22
+env.109.ssh.user=shell
+env.109.ssh.pwd=PASSWORD
+
+env.110.ssh.host=192.168.1.110
+env.110.ssh.port=22
+env.110.ssh.user=shell
+env.110.ssh.pwd=PASSWORD
+
+env.111.ssh.host=192.168.1.111
+env.111.ssh.port=22
+env.111.ssh.user=shell
+env.111.ssh.pwd=PASSWORD
+
+env.112.ssh.host=192.168.1.112
+env.112.ssh.port=22
+env.112.ssh.user=shell
+env.112.ssh.pwd=PASSWORD
+
+env.113.ssh.host=192.168.1.113
+env.113.ssh.port=22
+env.113.ssh.user=shell
+env.113.ssh.pwd=PASSWORD
+
+env.114.ssh.host=192.168.1.114
+env.114.ssh.port=22
+env.114.ssh.user=shell
+env.114.ssh.pwd=PASSWORD
+
+scenario=${HOME}/cubrid-testcases-private-ex/shell
+test_continue_yn=false
+cubrid_download_url=http://127.0.0.1/REPO_ROOT/store_02/10.1.0.6876-f9026f8/drop/CUBRID-10.1.0.6876-f9026f8-Linux.x86_64.sh
+testcase_exclude_from_file=${HOME}/cubrid-testcases-private-ex/shell/config/daily_regression_test_excluded_list_linux.conf
+testcase_update_yn=true
+testcase_git_branch=develop
+testcase_timeout_in_secs=604800
+test_platform=linux
+test_category=shell
+testcase_exclude_by_macro=LINUX_NOT_SUPPORTED
+testcase_retry_num=0
+delete_testcase_after_each_execution_yn=false
+enable_check_disk_space_yn=true
+
+owner_email=Mandy<cui.man@navercorp.com>
+
+git_user=cubridqa
+git_email=dl_cubridqa_bj_internal@navercorp.com
+git_pwd=N6P0Sm5U7h
+
+feedback_type=database
+
+feedback_db_host=192.168.1.86
+feedback_db_port=33080
+feedback_db_name=qaresu
+feedback_db_user=dba
+feedback_db_pwd=
+
+feedback_notice_qahome_url=http://192.168.1.86:8080/qaresult/shellImportAction.nhn?main_id=<MAINID>
+```
+shell_template.conf will be copied to \~/CTP/conf/shell_runtime.conf when test is started.  
+
+### Set ~/.bash_profile 
+*~/.bash_profile*  
+```
+# .bash_profile
+
+# Get the aliases and functions
+if [ -f ~/.bashrc ]; then
+        . ~/.bashrc
+fi
+
+# User specific environment and startup programs
+
+PATH=$JAVA_HOME/bin:$HOME/CTP/common/script:$PATH:$HOME/.local/bin:$HOME/bin
+
+export PATH
+
+export CTP_BRANCH_NAME="develop"
+export CTP_SKIP_UPDATE=0
+```
+
+### Create a script to start consumer
+~/start_test.sh
+```
+nohup start_consumer.sh -q QUEUE_CUBRID_QA_SHELL_LINUX -exec run_shell &
+```
+Execute the script to start listening the test message after deployment. This will start a shell test when the consumer receive the test message.
+```
+cd ~
+sh start_test.sh

```

Comment:
Only create `~/start_test.sh` file. Do not execute the script in here step.
When all of deployment is done, we will start the script.